cmd/vic-machine/validate/validator.go

```diff
@@ -301,10 +301,15 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 	}
 
 	// ensure gateway is populated
-	gateway, ok := input.MappedNetworksGateway["bridge"]
+	// FIXME: The gateway for the bridge network
+	//        should be the gateway address of the
+	//        default bridge network. The bridge
+	//        network pool should be specified
+	//        on the vic-machine command line.
+	gateway, ok := input.MappedNetworksGateways["bridge"]
 	if !ok {
-		ip, mask, _ := net.ParseCIDR("172.16.0.1/24")
-		gateway = &net.IPNet{
+		ip, mask, _ := net.ParseCIDR("172.16.0.1/16")
```

Comment:
I only discovered this recently. There is a netlink.ParseIPNet that will preserve the IP address in the resulting net so we could use this directly without lines 312-315